labs/SettingsControls/src/SettingsCard/SettingsCard.Properties.cs

```diff
@@ -47,16 +66,38 @@ public partial class SettingsCard : ButtonBase
 
 
     /// <summary>
-    /// Gets or sets an example string. A basic DependencyProperty example.
+    /// The backing <see cref="DependencyProperty"/> for the <see cref="Header"/> property.
     /// </summary>
-    public string Header
+    public static readonly DependencyProperty ContentAlignmentProperty = DependencyProperty.Register(
+        nameof(ContentAlignment),
+        typeof(ContentAlignment),
+        typeof(SettingsCard),
+        new PropertyMetadata(defaultValue: ContentAlignment.Right, (d, e) => ((SettingsCard)d).OnContentAlignmentPropertyChanged((ContentAlignment)e.OldValue, (ContentAlignment)e.NewValue)));
+
+    /// <summary>
+
+    /// <summary>
+    /// The backing <see cref="DependencyProperty"/> for the <see cref="WrapThreshold"/> property.
+    /// </summary>
+    public static readonly DependencyProperty WrapThresholdProperty = DependencyProperty.Register(
```

Comment:
I think we should start this as a resource key value for now?

Reply:
Done!